avatar/avatar.css

```diff
@@ -0,0 +1,26 @@
+@import url(/style.css);
+
+.avatar{
+    display: flex;
+    align-items: center;
+    margin: 5px;
+}
+.avatar-circular{
+    border-radius: 50%;
+    padding: 5px;
+}
+.avatar-s{
+    height: 2rem;
+    width: 2rem;
+}
+.avatar-m{
+    height: 3.5rem;
+    width: 3.5rem;
+}
+.avatar-l{
+    height: 5rem;
+    width: 5rem;
+}
+.avatar.square-avatar > * {
+    padding: 5px;
+}
```

Comment:
what are you intending to with this style?

Reply:
wanted to give some space around each avatar. Any other better way to do it?